api-reference/10 UI Components/dxDropDownBox/1 Configuration/items.md

```diff
@@ -1,10 +1,19 @@
 ---
 id: dxDropDownBox.Options.items
-hidden: 
+type: Array<Object>
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+An array of items used to synchronize the DropDownBox with an embedded UI component.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+The **DropDownBox** uses other UI components to render the **items**. Use the [contentTemplate](/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#contentTemplate) to embed a UI component such as the [List](/Documentation/ApiReference/UI_Components/dxList/) or the [TreeView](/Documentation/ApiReference/UI_Components/dxTreeView/). In this case, the **items** are displayed in accordance with the configuration and functionality of the embedded UI component.
+
+If you need to update the UI component items, reassign the entire **items** array as shown in the following example:
+
+    <!--JavaScript-->
+    dropDownBoxInstance.option('items', newItems);
+
+As an alternative to **items**, you can use the [dataSource](/api-reference/10%20UI%20Widgets/DataExpressionMixin/1%20Configuration/dataSource.md '/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#dataSource') property. It accepts the [DataSource](/api-reference/30%20Data%20Layer/DataSource '/Documentation/ApiReference/Data_Layer/DataSource/') object, whose underlying [stores](/concepts/70%20Data%20Binding/5%20Data%20Layer/1%20Creating%20DataSource/3%20What%20Are%20Stores.md '/Documentation/Guide/Data_Binding/Data_Layer/#Creating_DataSource/What_Are_Stores') provide an [API](/concepts/70%20Data%20Binding/5%20Data%20Layer/3%20Data%20Modification '/Documentation/Guide/Data_Binding/Data_Layer/#Data_Modification') that allows you to update individual items without reassigning the entire item collection. 
```

Comment:
- "As an alternative to **items**, you can use the [dataSource]" Here I would say 'You can use the [dataSource] instead of  **items**
- "object, whose underlying [stores]" Why do you use a comma here?
- "underlying [stores] provide an [API]" What do you mean when you say **provide**? Includes? 
- "an [API]" You do not need an article here

Reply:
1) agreed, will edit the text
2) will remove the comma
3) **provide** is a common term used in reference to APIs. This [StackOverflow post] (https://stackoverflow.com/questions/25178334/what-does-it-mean-to-provide-an-api#:~:text=To%20provide%20an%20API%20means,that%20is%20up%20to%20you.) might clarify that for you. (this one is specific to Web API)
4) Why shouldn't we use an article here?

Reply:
3. We've got a guideline regarding **provide**
4. This is how it is commonly used. There have also been discussions in our company about article usage and API. You are also referring to Application Programming Interface in general: not any API, not specific API, just API in general.